tools/ot/src/main/java/dk/alexandra/fresco/tools/ot/otextension/CoteReceiver.java

```diff
@@ -0,0 +1,134 @@
+package dk.alexandra.fresco.tools.ot.otextension;
+
+import dk.alexandra.fresco.framework.network.Network;
+import dk.alexandra.fresco.framework.util.Drbg;
+import dk.alexandra.fresco.framework.util.PaddingAesCtrDrbg;
+import dk.alexandra.fresco.framework.util.Pair;
+import dk.alexandra.fresco.framework.util.StrictBitVector;
+import dk.alexandra.fresco.tools.ot.base.Ot;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Protocol class for the party acting as the receiver in an correlated OT with
+ * errors extension.
+ * 
+ * @author jot2re
+ *
+ */
+public class CoteReceiver extends CoteShared {
+  // Random messages used for the seed OTs
+  private final List<Pair<StrictBitVector, StrictBitVector>> seeds;
```

Comment:
Unused?

Reply:
It has been removed as part of the seed OT refactoring. I am not sure exactly which commit.